Dockerfile.windows

```
@@ -0,0 +1,108 @@
+    refreshenv;
+
+# Install additional VS components that --includeRecommended doesn't include in containers
+RUN & 'C:\Program Files (x86)\Microsoft Visual Studio\Installer\vs_installer.exe' modify `
+      --installPath 'C:\Program Files\Microsoft Visual Studio\2022\Community' `
+      --add Microsoft.VisualStudio.Component.Windows11SDK.22621 `
+      --add Microsoft.VisualStudio.Component.Windows10SDK `
+      --quiet --norestart --wait; `
+    refreshenv;
+
+# Reload environment and add SDK bin to PATH
+RUN $sdkBin = Get-ChildItem 'C:\Program Files (x86)\Windows Kits' -Recurse -Filter 'rc.exe' -ErrorAction SilentlyContinue | `
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)